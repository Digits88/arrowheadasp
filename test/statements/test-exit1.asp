Test Exit statements.
<br>
Exit Do<br>
<br>
Do While 1  <% Do While 1 %><br>
    In Loop, Before Exit<br>
    Exit Do <%  Exit Do %><br>
    In Loop, After Exit<br>
Loop        <% Loop %><br>
<br>
Exit For<br>
<br>
For i = 1 to 100    <% For i = 1 to 100 %><br>
    In Loop, Before Exit<br>
    Exit For        <% Exit For %><br>
    In Loop, After Exit<br>
Next                <% Next %><br>
<br>
Exit Function<br>
<br>
Function MyFunc     <% Function MyFunc %><br>
    In Function, Before Exit<br>
    Exit Function   <%  Exit Function %>
    In Function, After Text<br>
End Function        <% End Function %><br>
<br>
MyFunc <% MyFunc %><br>
<br>
Exit Sub<br>
<br>
Sub MySub      <% Sub MySub %><br>
    In Sub, Before Exit<br>
    Exit Sub    <%  Exit Sub %>
    In Sub, After Text<br>
End Sub         <% End Sub %><br>
<br>
MySub <% MySub %><br>
