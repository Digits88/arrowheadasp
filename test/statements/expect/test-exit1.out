Test Exit statements.
<br>
Exit Do<br>
<br>
Do While 1  <br>
    In Loop, Before Exit<br>
    Exit Do <br>
<br>
Exit For<br>
<br>
For i = 1 to 100    <br>
    In Loop, Before Exit<br>
    Exit For        <br>
<br>
Exit Function<br>
<br>
Function MyFunc     <br>
<br>
MyFunc <br>
    In Function, Before Exit<br>
    Exit Function   <br>
<br>
Exit Sub<br>
<br>
Sub MySub      <br>
<br>
MySub <br>
    In Sub, Before Exit<br>
    Exit Sub    <br>
